docs/users/navigation/joysticks.md

```diff
@@ -0,0 +1,129 @@
+
+## Xbox Controller
+The navigation using an Xbox or PS4 controller are very similar, the only difference is the layout of the controllers. Otherwise the functionality is the same. The image below shows a map over the buttons and joysticks on a Xbox controller.
+
+![](images/xbox.png)
+
+The navigation using a Xbox controller in OpenSpace is defined in the Xbox asset file <code>(xbox.asset)</code> or the Xbox wireless asset file <code>(xbox-wireless.asset)</code>. The table below will give an overview of what each button or joystick on the Xbox controller does in OpenSpace. NA in the table specifies that this button or joystick has no functionality in the default version of OpenSpace.
+
+| Button or joystick | Description |
+|--------------------|-------------|
+| A | Toggle zoom friction |
+| B | Toggle rotation friction |
+| X | Switch focus to Earth |
+| Y | Switch focus to Mars |
+| LB | Pressed: Switch to local roll mode. <br/>Released: Switch back to normal mode |
+| LT | Zoom out |
+| RB | Pressed: Switch to global roll mode. <br/>Released: Switch back to normal mode |
+| RT | Zoom in |
+| Up | NA |
+| Right | NA |
+| Left | Toggle roll friction |
+| Down | NA |
+| Left Joystick up/down | Orbit around focus up/down |
+| Left Joystick left/right | Orbit around focus left/right |
+| Left Joystick Press | NA |
+| Right Joystick up/down | Pan camera up/down |
+| Right Joystick left/right | Pan camera left/right <br/>If LB or RB is pressed: Roll camera |
+| Right Joystick Press | NA |
+| Select | NA |
+| Start | NA |
+
+The LB and RB buttons switches the roll mode to local or global respectivly. The difference between these two is that the local roll mode rolls the camera around the center of the screen, while the global roll mode rolls the camera around the current focus.
+
+## PS4
+The image below shows a map over the buttons and joysticks on a PS4 controller.
+
+![](images/ps4.png)
+
+The navigation using a PS4 controller in OpenSpace is defined in the PS4 asset file <code>(ps4.asset)</code>. The table below will give an overview of what each button or joystick on the PS4 controller does in OpenSpace. NA in the table specifies that this button or joystick has no functionality in the default version of OpenSpace.
```

Comment:
The navigation using a PS4 controller in OpenSpace is defined in the PS4 asset file (<code>ps4.asset</code>). The table below gives an overview of what each button or joystick on the PS4 controller does in OpenSpace. NA in the table specifies that this button or joystick has no functionality in the default version of OpenSpace.